tests/libtest/test_container.py

```diff
@@ -0,0 +1,36 @@
+"""
+Module to perform IOs with several weights
+"""
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.container import Container
+from ocs_ci.framework.testlib import libtest
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='class')
+def container(request):
+    # Create storage class
+    log.info("Creating RBD and FS PVC")
+    rbd_pvc = Container(cleanup=True)
+    fs_pvc = Container(interface_type=constants.CEPHFILESYSTEM)
+    rbd_pvc.create()
+    fs_pvc.create()
```

Comment:
Since creation of these objects is happening before you add `teardown` as a finalizer, if anything goes wrong here and causes the setup to fail it will not be cleaned up by the finalizer.

Reply:
It is just a test example for library test, I think one can separate out into its own fixture setups. Also,  I have the option cleanup=True, which will cleanup at the start of the test based on labels on objects.